WindowsServerDocs/get-started/common-troubleshooting-procedures-kms-dns.md

```diff
@@ -194,7 +194,7 @@ allow-update { any; };
 ```
 ## Manually assign a KMS host to a KMS client
 
-By default, the KMS clients use the automatic discovery process. According to this process, a KMS client queries DNS for a list of servers that have published _vlmcs SRV records within the membership zone of the client. DNS returns the list of KMS hosts in a random order. The client picks a KMS host and tries to establish a session on it. If this attempt works, the client caches the name of the KMS host and tries to use it for the next renewal attempt. If the session setup fails, the client randomly picks another KMS host. We highly recommend that you use the automatic discovery process.  
+By default, the KMS clients use the automatic discovery process. According to this process, a KMS client queries DNS for a list of servers that have published _vlmcs SRV records within the membership zone of the client. DNS returns the list of KMS hosts in a random order. The client picks a KMS host and tries to establish a session on it. If this attempt works, the client caches the name of the KMS host and tries to use it for the next renewal attempt. If the session setup fails, the client randomly picks another KMS host. We highly recommend that you use the automatic discovery process.
```

Comment:
@illfated While this is correct, and apparently this problem has been there for some time, could you fix the underscore causing the rest of the text to be in italics? The best way to do it is to encompass the _vlmcs SRV in back ticks, so `_vlmcs SRV`. It marks it as code (just a way around markdown code since escape characters sometimes fail).

Reply:
Ah, right. I forgot that detail. Thank you. Will do.